### DISCUSSION TITLE
Can't play audio file in assets folder

### DISCUSSION THREAD

**Discussion by ghosteye1** [quasarframework/quasar#8961] [General - Components / Directives / etc] [answered]
Small mp3 file exists on my assets/sounds folder. I tried following but getting 404 for that.

const audio = new Audio('~assets/sounds/swiftly-610.mp3')
audio.play()

**Comment by rstoenescu** [comment, marked as answer]
You're literally trying to access `/~assets/sounds/swiftly-610.mp3`. There's no one to parse and interpret the requested URL, as in the SFC template. So two options: use the /public folder OR:

```
// second option
import mp3File from 'assets/sounds/swiftly-610.mp3'
const audio = new Audio(mp3File)
audio.play()
```

**Reply by ghosteye1** [reply]
I tried the second option but didn't work. Still getting 404. Where is the /public folder in quasar?

**Reply by rstoenescu** [reply]
<project_root>/public

On the second approach: please check what mp3File has (console.log it). It should have the URL path to it. Might be in a "default" prop (mp3File.default). But remember it will only have the URL path in it. If Audio API requires a full URL, you'll need to use the window.location to construct it.
